.github/workflows/test.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: pipx install poetry
+      - run: pipx install poetry==1.8.5
```

Comment:
Merged https://github.com/ollama/ollama-python/pull/409 here instead of main accidentally 🙃